scripts/ingestion/parsers/snapshot_download.py

```diff
@@ -0,0 +1,26 @@
+import requests
+import gzip
+import datetime
+from os.path import exists
+
+def get_snapshot_bydate(date, ARCH):
+    for DFSG in ["main", "contrib", "non-free"]:
+        if exists(f"./ingestion/parsers/Packagelist_DUMP/{date}-{ARCH}-{DFSG}_Packages"):
+            continue # don't re-download
+        while True:
+            url = f"http://snapshot.debian.org/archive/debian/{date}/dists/stable/{DFSG}/binary-{ARCH}/Packages.gz"
```

Comment:
can we keep these fstrings as `UPERCASE` variables to help w/ readability/reuse/updating?